index.html

```diff
@@ -106,21 +129,189 @@ <h3>Terminology</h3>
+
+				<p>[=eBraille publications=] do not support [=remote resources=] [[epub-33]]. All publication resources
+					MUST be located in or below the [=root directory=], as defined in <a href="#fileset-structure"
+					></a>.</p>
+			</section>
+		</section>
+		<section id="ebrl-fileset">
+			<h2>eBraille file set</h2>
+
+			<section id="fileset-intro" class="informative">
+				<h3>Introduction</h3>
+
+				<p>The [=eBraille file set=] is a physical manifestation of the <a
+						data-cite="epub-33#sec-container-abstract">OCF abstract container</a> [[epub-33]]. EPUB 3 only
+					defines its file set in the abstract because those files are expected to be zipped in the [=EPUB
+					container=] [[epub-33]]; the standard is not concerned with the physical files before they are
+					zipped or after they are unzipped. As a ZIP file does not have a true file system within it, the
+					rules for file naming and placement can only be defined virtually.</p>
+
+				<p>eBraille moves the rules for the OCF abstract container to the physical file set that exists before
+					and after packaging in the EPUB container. This allows an [=eBraille publication=] to be independent
+					of packaging, making it a format that can flexibly move between web deployment and end-user
+					distribution.</p>
+
+				<p>eBraille defines additional rules on the file structure in order to make it easier to read an
+					eBraille publication on the web or from a local file system. In particular, it requires the [=EPUB
+					navigation document=] [[epub-33]] be in the root of the file set and named <code>index.html</code>.
+					These requirements do not conflict with being able to package an eBraille publication as a valid
+					[=EPUB publication=] [[epub-33]], however.</p>
+			</section>
+
+			<section id="fileset-structure">
+				<h3>File and directory structure</h3>
+
+				<p>The [=eBraille file set=] MUST have a single common root directory &#8212; the [=publication root=]
+					&#8212; for all the contents of the [=eBraille publication=].</p>
+
+				<p>Unlike EPUB 3, the eBraille file set MUST NOT reference resources outside the publication root (i.e.,
+					[=remote resources=] [[epub-33]] are not supported).</p>
+
+				<p>The eBraille file set MUST contain the following files for compatibility with EPUB 3:</p>
+
+				<ul>
+					<li>The <a data-cite="epub-33#sec-zip-container-mime"><code>mimetype</code> file</a> [[epub-33]] in
+						the publication root.</li>
+					<li>The <a data-cite="epub-33#sec-container-metainf-container.xml"><code>container.xml</code>
+							file</a> under the <a data-cite="epub-33#sec-container-metainf"><code>META-INF</code>
+							directory</a> [[epub-33]].</li>
+				</ul>
+
+				<p>In addition, it MUST contain the following files in the publication root:</p>
+
+				<ul>
+					<li>The <a data-cite="epub-33#sec-nav">EPUB navigation document</a> [[epub-33]]. This file MUST be
+						named <code>index.html</code></li>
```

Comment:
shouldn't this be index.xhtml? (if so there are a few mentions to index.html.)

Reply:
index.xhtml likely won't work everywhere as a default index page on the web. You'd then have to configure the directory to accept it.

EPUB doesn't recommend extensions anymore, so using .html is fine. So far as I remember, the only thing using .xhtml does is force xhtml rendering in some browsers when opened from your file system.